client/dashboard/sites/index.tsx

```diff
@@ -79,23 +78,9 @@ const DEFAULT_FIELDS: Field< Site >[] = [
 		label: __( 'Subscribers' ),
 	},
 	{
-		id: 'backups',
-		type: 'boolean',
-		label: __( 'Backups' ),
-		getValue: ( { item } ) => !! item.plan?.features?.active?.includes( 'backups' ),
-		elements: [
-			{ value: true, label: __( 'Enabled' ) },
-			{ value: false, label: __( 'Disabled' ) },
-		],
-		render: ( { item } ) =>
-			item.plan?.features?.active?.includes( 'backups' ) ? (
-				<Icon icon={ check } />
-			) : (
-				__( 'Disabled' )
-			),
-		filterBy: {
-			operators: [ 'is' as Operator ],
-		},
+		id: 'backup',
+		label: __( 'Backup' ),
+		render: ( { item } ) => <LastBackup site={ item } />,
```

Comment:
we have to disable the sorting.

Reply:
Oh yes, updated in c7ae7e1cb57c91668d8551db453da812444d88c0.